ui/src/app/utils/runtime-utils.tsx

```diff
@@ -111,13 +111,14 @@ export interface AnalysisConfig {
   computeType: ComputeType;
   machine: Machine;
   diskConfig: DiskConfig;
-  // This should only be populated if !diskconfig.detachable.
+  // This should only be populated if !diskconfig.detachable. (What does this mean?)
```

Comment:
I updated the comment here, so I could ask about what this means.

Firstly, is a detachable disk a persistent disk?

DiskConfig has a property called detachable. I would imagine that this property designates whether a disk is detachable/persistent.

If that is the case, what is the detachedDisk mentioned below? The comment above says that it should only be set if diskConfig is not detachable.

Reply:
After talking about this as a team, I created a spike. I will update the comment with reference to the spike.